src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,309 @@
+package duke;// import libraries here
+import duke.Deadline;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Set;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+    private static ArrayList<String> tasks = new ArrayList<>();
+    private static ArrayList<Boolean> dones = new ArrayList<>();
+    private static ArrayList<String> types = new ArrayList<>();
+    private static ArrayList<String> dates = new ArrayList<>();
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        greeting();
+        loadTasks();
+        getCommand();
+    }
+
+    private static void getCommand() {
+        String input;
+        Scanner in = new Scanner(System.in);
+        // get user input
+        input = in.nextLine();
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
+                showList();
+                input = in.nextLine();
+                continue;
+            }
+            if (input.startsWith("done")) {
+                int number = Integer.parseInt(input.substring(input.length() - 1));
```

Comment:
Create a separate variable for input.substring(input.length() - 1) to make the code more readable